aioxmpp/muc/service.py

```diff
@@ -39,6 +39,21 @@
 from . import xso as muc_xso
 
 
+def _extract_one_pair(body):
```

Comment:
(Note: One should not use sorted but extract the minimal element, since this is O(m) vs. O(m·log(m)))